app/javascript/angular/code/services/_draw_session.js

```diff
@@ -1,31 +1,19 @@
 import _ from "underscore";
 import { drawNotes } from "../../../javascript/note";
 import * as Session from "../../../javascript/values/session";
-import { locationMarkersByLevel } from "../../../javascript/theme";
 
 export const drawSession = (sensors, map, heat, empty) => {
   var DrawSession = function() {};
 
   DrawSession.prototype = {
-    drawMobileSession: function(session, drawSessionStartingMarker) {
+    drawMobileSession: function(session) {
       if (!session || !sensors.anySelected()) {
         return;
       }
 
-      const startingMarker = drawSessionStartingMarker(session);
-
-      var suffix = " " + session.unit_symbol;
       var points = [];
 
-      this.measurements(session).forEach(function(measurement, idx) {
-        const marker = createMeasurementMarker(
-          measurement,
-          idx,
-          heat,
-          map,
-          suffix
-        );
-
```

Comment:
I don't think you can delete anything from this method - the selected sessions are not drawn now.